zerver/webhooks/taiga/view.py

```diff
@@ -271,13 +277,16 @@ def parse_message(message: Mapping[str, Any]) -> List[Dict[str, Any]]:
 
     return events
 
+
```

Comment:
Again, in all of our webhooks code base, functions are not separated by two new lines, only one. So we should stick to that standard for now! :) As a general rule, it is better to only include changes in a commit that are relevant to the particular commit.